addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -373,15 +393,47 @@ class AccountTax(models.Model):
                     'l10n_it_kind_exoneration',
                     'l10n_it_law_reference',
                     'amount',
-                    'l10n_it_vat_due_date')
+                    'children_tax_ids')
     def _check_exoneration_with_no_tax(self):
         for tax in self:
             if tax.l10n_it_has_exoneration:
                 if not tax.l10n_it_kind_exoneration or not tax.l10n_it_law_reference or tax.amount != 0:
                     raise ValidationError(_("If the tax has exoneration, you must enter a kind of exoneration, a law reference and the amount of the tax must be 0.0."))
-                if tax.l10n_it_kind_exoneration == 'N6' and tax.l10n_it_vat_due_date == 'S':
-                    raise UserError(_("'Scissione dei pagamenti' is not compatible with exoneration of kind 'N6'"))
+                if tax.l10n_it_kind_exoneration == 'N6' and tax._l10n_it_is_split_payment():
+                    raise UserError(_("Split Payment is not compatible with exoneration of kind 'N6'"))
+
+    def _l10n_it_is_split_payment(self):
+        """ Split payment means that the Public Administration buyer will pay VAT
+            to the tax agency instead of the vendor
+        """
+        self.ensure_one()
+
+        tax_tags = self.get_tax_tags(is_refund=False, repartition_type='tax')
+        if not tax_tags:
+            return False
+
+        it_tax_report_ve38_lines = self.env['account.report.line'].search([
+            ('report_id.country_id.code', '=', 'IT'),
+            ('code', '=', 'VE38'),
+        ])
+        if not it_tax_report_ve38_lines:
+            return False
+
+        ve38_lines_tags = it_tax_report_ve38_lines.expression_ids._get_matching_tags()
+        return bool(tax_tags & ve38_lines_tags)
+
+    def _l10n_it_get_split_payment_tax(self):
+        self.ensure_one()
+        if not self.children_tax_ids:
+            return False
+        return self.children_tax_ids.filtered(lambda t: t._l10n_it_is_split_payment()) or False
+
+    def _l10n_it_get_tax_kind(self):
+        """ Will be overridden by submodules implementing different localized kinds of taxes. """
+        if self._l10n_it_is_split_payment():
+            return 'split_payment'
```

Comment:
do we need a special type here?  Because it does not seem to be treated differently than the vat ones?

Reply:
![image](https://github.com/odoo/odoo/assets/1665365/14df1caf-999c-4998-8bec-06bb838503df)
It is... we have to detect if current tax is the split payment one, because otherwise we'll try to export the group of taxes, and that has 0% amount, so we need to swap it the group of taxes with the children tax, to export it correctly